README.md

```diff
@@ -35,6 +36,10 @@ Lint checks and auto-fixes will be run automatically on commit.
 
 New components can be added by executing `yarn add-component <ComponentName>`. The name of the component should be written using PascalCase. This will generate all important files for you in the correct location, and also update the index file for exporting the component. The generated code includes some `TODO` statements that you should fix.
 
+#### Adding new dependencies
+
+When adding new dependencies, you should also add that dependency to the `external` array in `rollup.config.js`. This is done to avoid having the dependency being part of the bundle.
```

Comment:
Can't we just reference `dependencies` from `package.json` in this file instead? That's the way I've solved that problem before. I'll create a separate issue for it.

Reply:
Added comment in #62 instead

Reply:
That seems like a great idea - lets follow up in 62.